### ISSUE TITLE
Registration issue

### ISSUE THREAD

**Issue by fragiledream** [neochapay/QtInstagram#4]
It seems after 10th April update, IG is returning HTTP 400 error when trying to register any account using your API. The account is probably getting created but instantly banned as when I try to create another with the same username, it says that it already exists. I tried with different IPs but the result is same. Account registration is working fine when done manually through their app or website.

**Comment by neochapay** [comment]
You can create account on that API but you must end registration on site

**Comment by fragiledream** [comment]
Pardon me but what do you mean by "end registration on site"?

**Comment by neochapay** [comment]
login to insragram.com and confurm

**Comment by fragiledream** [comment]
Yeah but the problem is that after 10th April, no accounts are being created so there is no question of confirmation. If I do it manually from mobile or browser, it works, but using your API, it returns HTTP Error 400 saying that the request is spam and "error_type" is "signup_block". But the same IP can be used for registering from browser/mobile.
